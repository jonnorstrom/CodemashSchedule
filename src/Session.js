import React, { Component } from 'react';

class Session extends Component {
  render() {
    return (
      <div>
        <p>{this.props.title}</p>
        <p>{this.props.abstract}</p>
      </div>
    )
  }
}

export default Session
