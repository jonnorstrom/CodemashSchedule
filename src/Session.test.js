import React from 'react';
import renderer from 'react-test-renderer'
import Session from './Session';

it('renders without crashing', () => {
  const output = renderer.create(<Session title="a" abstract="b"/>);

  expect(output).toMatchSnapshot()
});
