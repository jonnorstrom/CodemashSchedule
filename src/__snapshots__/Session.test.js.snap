exports[`test renders without crashing 1`] = `
<div>
  <p>
    a
  </p>
  <p>
    b
  </p>
</div>
`;
